CHANGELOG.md

```diff
@@ -4,9 +4,13 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+## 0.11.0 (2022/09/19)
+
 ### Bug Fixes
 
-* Fix bug preventing delete of org. ([PR](https://github.com/hashicorp/boundary/pull/2465))
+* Fix bug preventing delete of org ([PR](https://github.com/hashicorp/boundary/pull/2465))
+* Show user warning log when token period is less than jobRunInterval ([PR](https://github.com/hashicorp/boundary/pull/2446))
+* Return an error for vault tokens that will expire before the scheduler can run ([PR](https://github.com/hashicorp/boundary/pull/2444))
```

Comment:
I am not sure either of these need to be in the changelog

Reply:
thanks!

Reply:
Hmm I did a multi select but it does not seem to be rendering, this comment was talking about the last two entries 

```
* Show user warning log when token period is less than jobRunInterval ([PR](https://github.com/hashicorp/boundary/pull/2446))
* Return an error for vault tokens that will expire before the scheduler can run ([PR](https://github.com/hashicorp/boundary/pull/2444))
```